### DISCUSSION TITLE
Mobile Playability

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#13317] [Q&A] [answered]
std: how are you supposed to play streams? is it possible to implement tapping anywhere on the screen registers as a click and you can just point?

taiko: is there only binding to center drum (which is just regular taps)? wouldn't it make more sense to simulate a taiko drum at the bottom of the screen (which would just be the empty whitespace divided into three sections, left and right being rim and a larger middle being center)?

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/4201, tracking for all rulesets.
